src/Balloon.java

```diff
@@ -0,0 +1,41 @@
+
+
+public class Balloon extends Enemy{
+
+	public Balloon(Level l) {
+		image = "enemy111.png";
+		alive = true;
+
+		do {
+			xPos = (int)(Math.random()*(Constants.BOARD_SIZE-1)+1);
+			yPos = (int)(Math.random()*(Constants.BOARD_SIZE-1)+1);
+
+		} while (l.board[xPos][yPos].available == false || (xPos == 1 && yPos == 1) );

```

Comment:
it will keep looking for a random position that is available and which position is not (1,1)